loops-and-arrays.js

```diff
@@ -6,6 +6,18 @@
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
 
+console.log("originele waardes:");
+for (let i = 0; i < names.length; i++) {
```

Comment:
Dit had eenvoudiger gekund door een console.log(names), zoals ook in de uitwerkingen